### PR TITLE
Added FAQ about 3 minute override to merge

### PR DESCRIPTION
**Pull request by WhiteNoiseDev**

### CODE REVIEW

**Review by marcusolsson — COMMENTED** [review]
I have a few suggestions:

- Remove "!?!". I assume this is to empathize with the user, but also risk reinforcing a negative feeling. I'd recommend that we keep a neutral tone.
- I think this explanation risks being overly technical and can be simplified.
- We should avoid putting specific numbers (3 minutes) for technical implementations. The team might adjust the time down the road, and there's really no need for users to know the exact timeout.

For example, here's a suggestion with fewer technical terms that I think still gets the point across. Let me know if not! 🙏 

> ## Obsidian Sync deleted a note I just created on two devices
> 
> Generally, Obsidian Sync tries to [[#Conflict resolution|resolve conflicts]] between devices by merging the content of the conflicting notes.
> 
> Unfortunately, merging conflicting notes can cause issues for users who open automatically generated daily notes on startup. To address this, Obsidian Sync adds an exception for when two notes were created on separate devices within a few minutes from each other.
> 
> In this case, Obsidian Sync keeps the first note without merging the second one. You can still recover the second note using [[File recovery]].

**Comment by Akrson** [comment]
docker pull ghcr.io/microsoft/opensource.microsoft.com/custom-jekyll-builder:sha-2415822

File: en/Obsidian Sync/Troubleshoot Obsidian Sync.md [hunk 1]
```diff
@@ -12,6 +12,13 @@ When Sync downloads a new version of a file, and finds that there are conflicts
 > To find when conflicts have happened, you can search for "Merging conflicted file" in **Settings** → **Sync** → **Sync activity** → **View**.
 
 
+## Obsidian Sync deleted a note I just created in two devices!?!
+Generally, Obsidian merges the content of notes in different devices.
```
Comment (liamcain) [inline en/Obsidian Sync/Troubleshoot Obsidian Sync.md:16]:
```suggestion
## Obsidian Sync deleted a note I just created in two devices!?!

Generally, Obsidian merges the content of notes in different devices.
```

**Comment by WhiteNoiseDev** [comment]
Ok, I accepted both feedbacks.

**Pull request merged**